compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -498,7 +498,7 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
     // unrelated change.
```

Comment:
I am against merging this as is. I believe the original version is both clearer and more efficient. Computing `|` on the fly as a cost, that's why we created all these flag set. Doing several flag tests in a row also has a cost. On the other hand, testing against a FlagConjunction with `is` is fast and does not require boxing.